docs/rules/no-warning-comments.md

```diff
@@ -1,33 +1,27 @@
 # Disallow Warning Comments (no-warning-comments)
 
-Often code is marked during development process for later work on it or with additional thoughts. Examples are typically `// TODO: do something` or `// FIXME: this is not a good idea`. These comments are a warning signal, that there is something not production ready in your code. Most likely you want to fix it or remove the comments before you roll out your code with a good feeling.
+Developers often add comments to code which is not complete or needs review. Most likely you want to fix or review the code, and then remove the comment before you roll out your code with a good feeling.
 
-## Rule Details
+```js
+// TODO: do something
+// FIXME: this is not a good idea
+```
 
-This rule can be used to help finding these `warning-comments`. It can be configured with an array of terms you don't want to exist in your code. It will raise a warning when one or more of the configured `warning-comments` are present in the checked files.
+## Rule Details
 
-The default configuration looks like this:
+This rule reports comments that warn something is not production ready in your code.
```

Comment:
Eee... I think this might be a bit too vague:
```js
// DO NOT PUSH THIS CODE TO PROD!!!
```
will not be flagged, but the user might expect it. I would phrase it something like:
This rule reports comments that include predefined terms specified in configuration.